firstvoices/backend/resources/pages.py

```diff
@@ -40,15 +40,23 @@ def before_import_row(self, row, **kwargs):
             created_by_user = self.get_user_or_none(row["created_by"])
             site = Site.objects.get(id=row["site"])
             widgets_list = row["widgets"].split(",")
-            site_widget_list = SiteWidgetList.objects.create(site=site)
+            site_widget_list = SiteWidgetList.objects.create(
```

Comment:
When running locally I get warnings for each SiteWidgetList and SiteWidgetListOrder model created in this `before_import_row` function.

e.g.  RuntimeWarning: DateTimeField SiteWidgetList.last_modified received a naive datetime (2022-09-12 14:38:27.007000) while time zone support is active.

Would it be worth it to convert these date values to proper ones including a timezone or can the warnings be ignored? CC: @caforbes

Reply:
Thes warnings occur in both the page import and site_homepage_widget import

Reply:
Good catch! The timestamps could probably be removed from the import and possibly added into other imports as a mixin later. @caforbes do you have any thoughts on this?

Reply:
After thinking about it, I don't think it is really important to preserve any created/last_modified data for this intermediate model which does not have a nuxeo correlate. It's completely fine if the created/last_modified dates default to the migration time.